src/stdune/sexp.ml

```diff
@@ -81,6 +81,11 @@ module Of_sexp = struct
     Printf.ksprintf (fun msg ->
       of_sexp_error loc ?hint ("No variables allowed " ^ msg)) fmt
 
+  let _fix f =
```

Comment:
Why not modify the type of `fix` instead of adding this function?

Reply:
This is just an artefact from the rebase. Removed.